src/test/java/org/apache/commons/collections4/trie/PatriciaTrieTest.java

```diff
@@ -243,28 +242,26 @@ public void testPrefixMap() {
         map = trie.prefixMap("Ab");
         Assertions.assertTrue(map.isEmpty());
         Assertions.assertEquals(0, map.size());
-        try {
-            final Object o = map.firstKey();
-            Assertions.fail("got a first key: " + o);
-        } catch(final NoSuchElementException nsee) {}
-        try {
-            final Object o = map.lastKey();
-            Assertions.fail("got a last key: " + o);
-        } catch(final NoSuchElementException nsee) {}
+
+        final SortedMap<String, String> map1 = map;
+        Assertions.assertThrows(NoSuchElementException.class, () -> map1.firstKey());
+
+        final SortedMap<String, String> map2 = map;
+        Assertions.assertThrows(NoSuchElementException.class, () -> map2.lastKey());
+
         iterator = map.values().iterator();
         Assertions.assertFalse(iterator.hasNext());
 
         map = trie.prefixMap("Albertooo");
         Assertions.assertTrue(map.isEmpty());
         Assertions.assertEquals(0, map.size());
-        try {
-            final Object o = map.firstKey();
-            Assertions.fail("got a first key: " + o);
-        } catch(final NoSuchElementException nsee) {}
-        try {
-            final Object o = map.lastKey();
-            Assertions.fail("got a last key: " + o);
-        } catch(final NoSuchElementException nsee) {}
+
+        final SortedMap<String, String> map3 = map;
+        Assertions.assertThrows(NoSuchElementException.class, () -> map3.firstKey());
+
+        final SortedMap<String, String> map4 = map;
+        Assertions.assertThrows(NoSuchElementException.class, () -> map4.lastKey());
```

Comment:
Maybe keep the comments above. I don't know what first or last key represent in this case, but I assume it could be useful to see that in the test failure output (I guess that's why the original developer put them there).

Reply:
I keep getting told to drop the failure message or not check the exception message, so assumed that is the apache commons standard. I can change, but it's slightly confusing as each apache commons pr review appears to have different criterial or standards that I need to follow.

Reply:
done, in next push